.gitignore

```diff
@@ -24,6 +25,8 @@ settings.py
 static/javascripts/metadata_tooltips.js
 ui_library
 ~$
+settings.py.local
+templates/info
```

Comment:
Is "templates/info" a file or directory? There is an entry "templates/info/" in the .gitignore file.

Reply:
It seems like the way to make the templates/info is a controversy.  Rushiraj mentioned that these had been moved into the main repository, but it seems like the deploy on the servers might still use the old separate repository.

I symlinked this in on my local machine to use the copy in the main repository like Rushiraj mentioned and my symlink shouldn't be committed since it causes deploy problems.